addons/mail/static/src/core/common/thread.js

```diff
@@ -430,11 +430,17 @@ export class Thread extends Component {
         return msg.datetime.ts - prevMsg.datetime.ts < 60 * 1000;
     }
 
+    /**
+     *
+     * @param {import("models").Message} previousMessage
+     */
     isNewMessageSeparatorVisible(previousMessage) {
```

Comment:
Nitpicking: the jsdoc could be a single line. Why have more LOCs when you can have less? It's nice to scroll less when looking / working on the file.

_(Of course I'm not saying to put everything in a single line... But when there's something that takes a single expression / trivial statement, the expected footprint should match and be ideally 1 LOC)_